RCAIDE/Library/Methods/Energy/Sources/Batteries/Common/__init__.py

```diff
@@ -10,7 +10,8 @@
 # ----------------------------------------------------------------------------------------------------------------------
 # @ingroup Methods-Energy-Sources-Battery
 
-from .append_battery_conditions             import append_battery_conditions 
+from .append_battery_conditions             import append_battery_conditions
```

Comment:
i think you can combine these two lines and import two functions at  once , also you can import everything using "*"

Reply:
done